testkit/Dockerfile

```diff
@@ -41,4 +39,12 @@ USER driver
 WORKDIR /home/driver
 CMD /bin/bash
 RUN mkdir /home/driver/.npm_global
+
+# Configuring NodeJS version
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+RUN nvm install ${NODE_VERSION:=10}
```

Comment:
probably for another PR, but Node 10 has reached EOL as you know

Reply:
It's end-of-life, but it is the one which you could install with apt in debian/ubuntu lts.

Reply:
(not important for this PR but:) for Go, I manually install a recent version, I am not installing the default, EOL one.